app/controllers/vehicles_controller.rb

```diff
@@ -16,4 +16,14 @@ def create
     redirect_to "/vehicles"
   end
 
+  def edit
+    @vehicle = Vehicle.find(params[:id])
+  end
+
+  def update
+    vehicle = Vehicle.find(params[:id])
+    vehicle.update(name:params[:name],passenger_capacity:params[:passenger_capacity],locked:params[:locked])
```

Comment:
I recommend using strong params here which you can see how to implement in the [passing data in rails lesson](https://backend.turing.io/module2/lessons/passing_data_in_rails).

Reply:
I would rather leave this as is until that lesson so that I understand what is going on and get some practice after building conceptual knowledge, but I will implement it then.

Reply:
Great that will reinforce the class. Good idea.